modules/xmpp/SampleSdpStrings.js

```diff
@@ -0,0 +1,162 @@
+"a=extmap:3 http://www.webrtc.org/experiments/rtp-hdrext/abs-send-time\r\n" +
+"a=setup:passive\r\n" +
+"a=mid:video\r\n" +
+"a=sendrecv\r\n" +
+"a=ice-ufrag:adPg\r\n" +
+"a=ice-pwd:Xsr05Mq8S7CR44DAnusZE26F\r\n" +
+"a=fingerprint:sha-256 6A:39:DE:11:24:AD:2E:4E:63:D6:69:D3:85:05:53:C7:3C:38:A4:B7:91:74:C0:91:44:FC:94:63:7F:01:AB:A9\r\n" +
+"a=ssrc:1757014965 msid:0836cc8e-a7bb-47e9-affb-0599414bc56d bdbd2c0a-7959-4578-8db5-9a6a1aec4ecf\r\n" +
+"a=ssrc:1757014965 cname:peDGrDD6WsxUOki/\r\n" +
+"a=ssrc:984899560 msid:0836cc8e-a7bb-47e9-affb-0599414bc56d bdbd2c0a-7959-4578-8db5-9a6a1aec4ecf\r\n" +
+"a=ssrc:984899560 cname:peDGrDD6WsxUOki/\r\n" +
+"a=ssrc-group:FID 1757014965 984899560\r\n" +
+"a=rtcp-mux\r\n";
+
+var simulcastRtxVideoMLineSdp = "" +
+"m=video 9 RTP/SAVPF 100 96\r\n" +
+"c=IN IP4 0.0.0.0\r\n" +
+"a=rtpmap:100 VP8/90000\r\n" +
+"a=fmtp:96 apt=100\r\n" +
+"a=rtcp:9 IN IP4 0.0.0.0\r\n" +
+"a=rtcp-fb:100 ccm fir\r\n" +
+"a=rtcp-fb:100 nack\r\n" +
+"a=rtcp-fb:100 nack pli\r\n" +
+"a=rtcp-fb:100 goog-remb\r\n" +
+"a=extmap:3 http://www.webrtc.org/experiments/rtp-hdrext/abs-send-time\r\n" +
+"a=setup:passive\r\n" +
+"a=mid:video\r\n" +
+"a=sendrecv\r\n" +
+"a=ice-ufrag:adPg\r\n" +
+"a=ice-pwd:Xsr05Mq8S7CR44DAnusZE26F\r\n" +
+"a=fingerprint:sha-256 6A:39:DE:11:24:AD:2E:4E:63:D6:69:D3:85:05:53:C7:3C:38:A4:B7:91:74:C0:91:44:FC:94:63:7F:01:AB:A9\r\n" +
+"a=ssrc:1757014965 msid:0836cc8e-a7bb-47e9-affb-0599414bc56d bdbd2c0a-7959-4578-8db5-9a6a1aec4ecf\r\n" +
+"a=ssrc:1757014965 cname:peDGrDD6WsxUOki/\r\n" +
+"a=ssrc:1479742055 msid:0836cc8e-a7bb-47e9-affb-0599414bc56d bdbd2c0a-7959-4578-8db5-9a6a1aec4ecf\r\n" +
+"a=ssrc:1479742055 cname:peDGrDD6WsxUOki/\r\n" +
+"a=ssrc:1089111804 msid:0836cc8e-a7bb-47e9-affb-0599414bc56d bdbd2c0a-7959-4578-8db5-9a6a1aec4ecf\r\n" +
+"a=ssrc:1089111804 cname:peDGrDD6WsxUOki/\r\n" +
+"a=ssrc:855213044 msid:0836cc8e-a7bb-47e9-affb-0599414bc56d bdbd2c0a-7959-4578-8db5-9a6a1aec4ecf\r\n" +
+"a=ssrc:855213044 cname:peDGrDD6WsxUOki/\r\n" +
+"a=ssrc:984899560 msid:0836cc8e-a7bb-47e9-affb-0599414bc56d bdbd2c0a-7959-4578-8db5-9a6a1aec4ecf\r\n" +
+"a=ssrc:984899560 cname:peDGrDD6WsxUOki/\r\n" +
+"a=ssrc:2963867077 msid:0836cc8e-a7bb-47e9-affb-0599414bc56d bdbd2c0a-7959-4578-8db5-9a6a1aec4ecf\r\n" +
+"a=ssrc:2963867077 cname:peDGrDD6WsxUOki/\r\n" +
+"a=ssrc-group:FID 1757014965 984899560\r\n" +
+"a=ssrc-group:FID 1479742055 855213044\r\n" +
+"a=ssrc-group:FID 1089111804 2963867077\r\n" +
+"a=ssrc-group:SIM 1757014965 1479742055 1089111804\r\n" +
+"a=rtcp-mux\r\n";
+
+var simulcastSdp = baseSessionSdp + baseAudioMLineSdp + simulcastVideoMLineSdp + baseDataMLineSdp;
+var simulcastRtxSdp = baseSessionSdp + baseAudioMLineSdp + simulcastRtxVideoMLineSdp + baseDataMLineSdp;
+var plainVideoSdp = baseSessionSdp + baseAudioMLineSdp + plainVideoMLineSdp + baseDataMLineSdp;
+var rtxVideoSdp = baseSessionSdp + baseAudioMLineSdp + rtxVideoMLineSdp + baseDataMLineSdp;
+
+exports.simulcastSdp = transform.parse(simulcastSdp);
```

Comment:
Why not export `export const simulcastSdp = transform.parse(simulcastSdp);`?

PS: The question is valid for the next lines also.